src/website/views/components/carousel/examples/_hideButtonControllers.pug

```diff
@@ -0,0 +1,47 @@
+h3 Hide Button Controllers
+p.-text
+  | Use the attribute <code>hideButtonControllers</code> to render Carousel without button controllers.
+.example.-mb--3
+  .-p--3
+    chi-carousel(hide-button-controllers)
+      .-d--flex(slot="items")
+        each item in [1, 2, 3, 4, 5, 6, 7, 8]
+          .chi-carousel__item.-p--1(style="width: 18.1rem; height: 7rem;")
+            .-w--100.-h--100.-bg--grey-10.-d--flex.-align-items--center.-justify-content--center= item
+  .example-tabs.-pl--2
+    ul.chi-tabs#example-accordion-base
+      li.-active
+        a(href=`#webcomponent-accordion-base`) Web Component
+      li.-disabled
+        a(href=`#html-accordion-base`) HTML Blueprint
+  .chi-tabs-panel.-active#webcomponent-accordion-base
+    :code(lang="html")
+      <chi-carousel hide-button-controllers>
```

Comment:
I would suggest adding the dot controllers to this example. We want to encourage developers to give users at least one way to navigate the carousel on non-touchscreen devices if no prev and next buttons are present.

```suggestion
    chi-carousel(hide-button-controllers dots)
      .-d--flex(slot="items")
        each item in [1, 2, 3, 4, 5, 6, 7, 8]
          .chi-carousel__item.-p--1(style="width: 18.1rem; height: 7rem;")
            .-w--100.-h--100.-bg--grey-10.-d--flex.-align-items--center.-justify-content--center= item
  .example-tabs.-pl--2
    ul.chi-tabs#example-accordion-base
      li.-active
        a(href=`#webcomponent-accordion-base`) Web Component
      li.-disabled
        a(href=`#html-accordion-base`) HTML Blueprint
  .chi-tabs-panel.-active#webcomponent-accordion-base
    :code(lang="html")
      <chi-carousel hide-button-controllers dots>
```